src/controllers/carpool.ctrl.ts

```diff
@@ -26,7 +26,8 @@ module CarpoolController {
       'campus': campusCtrl.toOutputJSON(carpool.campus),
       'participants': carpool.participants.map(userCtrl.toOutputJSON),
       'pickupLocation': carpool.pickupLocation,
-      'href': '/carpools/' + carpool.id
+      'href': '/carpools/' + carpool.id,
+      'dist': carpool['dist'] ? carpool['dist'] : 0
```

Comment:
I agree with what you said in slack about this default being something other that 0.  Could just make it a negative number since distances are never negative.